CMSgov/easi-app
# [EASI-2947] - Advice Letter Task Status for Admins

# EASI-2947

## Changes and Description

- Users will only see "Cannot Start Yet", "In Review", or "Complete" on the task list page for the advice letter step. Other views will still show the normal statuses.
- I noted that there are no tests for the task list or advice letter statuses, only the overall TRB status. I'm open for suggestions on how to proceed with covering the change with tests if necessary.

## How to test this change

First start and seed the app:

```sh
$ scripts/dev up
$ scripts/dev db:clean
$ scripts/dev db:seed
```

Then use local auth to login as user `ABCD` with TRB admin job codes, go to the TRB tab and view the TRB request task list pages. Admins should only see the three aforementioned statuses. You can also logout and log back in as the same user without admin job codes to verify you can still see the old statuses.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
